namespaces/collectd.yml

```diff
@@ -146,11 +156,6 @@ namespace:
       Corresponds to collectd's disk plugin.
       Or virt plugin with types:disk_octets/disk_ops.
     fields:
-    - name: disk_name
```

Comment:
Could we consider this from the consumer of the metrics?  If a person is trying to develop graphs or build queries, and they don't know `collectd`, wouldn't it be somewhat confusing to have to use the plugin instance field instead of an explicit name in the same metric subgroup?

Can we drop the `Plugin_instance` field entirely, and move its values to an appropriate identifier fields in each of the metrics subgroups?